runtime/JavaScript/src/antlr4/Lexer.d.ts

```diff
@@ -6,6 +6,9 @@ import {Token} from "./Token";
 export declare class Lexer extends Recognizer<number> {
 
     static DEFAULT_MODE: number;
+    static literalNames: (string | null)[];
```

Comment:
I'm not sure this makes any sense... these are defined with the generated XXXLexer, they don't exist in the base class.

Reply:
Removed in my last commit.